snailtrail/src/exploration/betweenness_centrality.rs

```diff
@@ -46,9 +46,32 @@ impl ScaleReduce for f64 {
 pub trait ExtendedData: Data + Eq + Hash + Copy + Debug {}
 impl<T: Data + Eq + Hash + Copy + Debug> ExtendedData for T {}
 
+/// Methods to push stream timestamps around
+pub trait PushTime<S: Scope<Timestamp = u64>, D: Data> {
```

Comment:
This looks like a fine solution to the way Snailtrail computes its metric, but it may have an performance impact. This comment is really just to keep in mind that it makes a performance comparison harder between what's published in the paper and what's still to come.

Reply:
I will keep it in mind, thanks! Perhaps then it will make sense for the benchmark to compare current SnailTrail vs. SnailTrail using timely 0.9 vs. SnailTrail w/ future metrics to consider such effects.